index.php

```diff
@@ -1,5 +1,6 @@
 <?php
-$db = new PDO('MySQL: host = 127.0.0.1;dbname=portfolioProject', 'root');
+
```

Comment:
copy everything from index.html into here

Reply:
and get rid of index.html

Reply:
Done, although still have html so I don't start breaking my site navigation just yet

Reply:
Get rid of unnecessary files.